### PR TITLE
Add email address in user account properties in documentation

### PR DESCRIPTION
**Pull request by ZeikoFr**
Explain the use of an email address in an account.

The picture should be also updated but idk how to do it.

### CODE REVIEW

File: admin_manual/configuration_user/user_configuration.rst [hunk 1]
```diff
@@ -45,6 +45,11 @@ User accounts have the following properties:
   The admin sets the new user's first password. Both the user and the admin can
   change the user's password at anytime.
 
+*Email address*
+  You can set an email address for a user.
+  This address can be used when you first set-up an account so the user receive a mail asking him to create a password.
```
Comment (MorrisJobke) [inline admin_manual/configuration_user/user_configuration.rst:50]:
```suggestion
  This address can be used when you first set-up an account so the user receive a mail asking them to create a password if none is provided.
```

Comment (MorrisJobke) [reply admin_manual/configuration_user/user_configuration.rst:50]: We prefer to use gender neutral language. And I added a comment that this is only done when no password is provided by the admin.

Comment (ZeikoFr) [reply admin_manual/configuration_user/user_configuration.rst:50]: Thanks !

Comment (tflidd) [reply admin_manual/configuration_user/user_configuration.rst:50]: If you could apply the suggestions and sign your commit, we can do a final review and merge your changes!

Comment (ZeikoFr) [reply admin_manual/configuration_user/user_configuration.rst:50]: > If you could apply the suggestions and sign your commit, we can do a final review and merge your changes!

Sorry i'm not used to merge to other repo, is this possible when I don't have forked repo anymore ? Or do I need to create a new PR ?

Comment (tflidd) [reply admin_manual/configuration_user/user_configuration.rst:50]: > Sorry i'm not used to merge to other repo, is this possible when I don't have forked repo anymore ? Or do I need to create a new PR ?

Not sure, I'm not a github expert. But considering the few changes, it is perhaps faster just doing a new PR.

**Comment by susnux** [comment]
This PR is stale for some time, access to branch is not granted for rebase, so I will close this PR.
Feel free to re-open if you want to continue with this.

**Pull request closed without merging**